src/components/PlusButtonEdge.tsx

```diff
@@ -1,18 +1,23 @@
 import './CustomEdge.css';
 import { BranchBuilder, MiniCatalog } from '@kaoto/components';
 import { usePosition, useShowBranchTab } from '@kaoto/hooks';
-import { StepsService, ValidationService, VisualizationService } from '@kaoto/services';
+import { StepsService, ValidationService } from '@kaoto/services';
 import { useIntegrationJsonStore } from '@kaoto/store';
 import { IStepProps, IVizStepNode } from '@kaoto/types';
 import { Popover, Tooltip } from '@patternfly/react-core';
 import { PlusIcon } from '@patternfly/react-icons';
 import { ReactNode } from 'react';
-import { EdgeText, getBezierPath, Position, useReactFlow } from 'reactflow';
+import { EdgeText, getBezierPath, Position } from 'reactflow';
 
 const foreignObjectSize = 40;
 
 export interface IPlusButtonEdge {
-  data?: any;
+  data?: {
```

Comment:
[curious-question]: What should be the right type for this component? I couldn't find a type combining `showBranchesTab` and `sourceStepNode` 🤔

Reply:
Ah, I see it now: https://github.com/KaotoIO/kaoto-ui/pull/1805/files#diff-3c58cb917b8929dfa554c2eca4f1e3cd89c20bc0b02d42cce794aa3f9b88d24f